packages/2d/src/components/Layout.ts

```diff
@@ -157,18 +158,32 @@ export class Layout extends Node {
   @initial(null)
   @signal()
   public declare readonly fontFamily: SimpleSignal<string | null, this>;
-  @initial(null)
   @signal()
-  public declare readonly fontSize: SimpleSignal<number | null, this>;
+  public declare readonly fontSize: SimpleSignal<number, this>;
+
+  protected computeFontSize(): number {
+    this.requestLayoutUpdate();
+    return Number(this.styles.getPropertyValue('font-size').slice(0, -2));
```

Comment:
You could use `parseFloat`:
```suggestion
    return parseFloat(this.styles.getPropertyValue('font-size'));
```